heka/files/lua/encoders/status_nagios.lua

```diff
@@ -19,8 +19,7 @@ local consts = require 'gse_constants'
 local lma = require 'lma_utils'
 local interp = require "msg_interpolate"
 
-local host = read_config('nagios_host')
-local service_template = read_config('service_template') or error('service_template is required!')
+local service = read_config('service') or 'member'
```

Comment:
Any reason for making this configurable? Do you actually use some other field that `member`?